### PR TITLE
docs.niteoweb.com moved to docs.niteo.co

### PR DESCRIPTION
**Pull request by zupo**
Refs niteoweb/operations#402

### CODE REVIEW

File: managing-docs.md [hunk 1]
```diff
@@ -6,9 +6,9 @@ We use the following services for storing information:
...
-- **[Docs](http://docs.niteoweb.com)** - Technical documentation for projects, generated automatically by Sphinx on every code change.
-- **[Books](http://books.niteoweb.com)** - Every time we buy a book, it goes into our digital library. Enjoy.
-- **[Videos](http://videos.niteoweb.com)** - A collection of videos of our talks on public and private events, conferences, meetups, IRLs, etc.
+- **[Docs](http://docs.niteo.co)** - Technical documentation for projects, generated automatically by Sphinx on every code change.
+- **[Books](http://books.niteo.co)** - Every time we buy a book, it goes into our digital library. Enjoy.
+- **[Videos](http://videos.niteo.co)** - A collection of videos of our talks on public and private events, conferences, meetups, IRLs, etc.
```
Comment (marbeann) [inline managing-docs.md]: "talks on public and..." should be "talks **in** public and..."

Comment (zupo) [reply managing-docs.md]: Really? I give a talk *in* an event, not *on* an event?

Comment (marbeann) [reply managing-docs.md]: Yes. On refers to the talk's subject. Like _I gave a talk **on** public speaking._"

Comment (zupo) [reply managing-docs.md]: But `in` sounds strange. `at`?

Comment (marbeann) [reply managing-docs.md]: `in` is used for general, `at` for specific. Like _"He gave a talk on public speaking at the Google event."_

Comment (zupo) [reply managing-docs.md]: Thanks for the grammar lesson ❤️

**Pull request merged**